support-services/docker-entrypoint.d/00-remove-es-watermark.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+curl -s -X PUT "elasticsearch:9200/_cluster/settings" -H 'Content-Type: application/json' -d'
+{
+  "transient": {
+    "cluster.routing.allocation.disk.watermark.low": "30mb",
+    "cluster.routing.allocation.disk.watermark.high": "20mb",
+    "cluster.routing.allocation.disk.watermark.flood_stage": "10mb",
+    "cluster.info.update.interval": "1m"
+  }
+}'
+
+echo "Removed disk space watermark in ElasticSearch."
```

Comment:
Does it remove or set `disk space watermark`?

Reply:
it essentially removes it by setting it to a very low minimum